packages/notify-client/src/types/client.ts

```diff
@@ -114,28 +114,28 @@ export declare namespace NotifyClientTypes {
     expiry: number;
   }
 
-  interface NotifyServerMessage {
+  interface NotifyServerNotification {
     title: string;
     sent_at: number;
     body: string;
     id: string;
-    url: string;
-    type?: string;
+    url?: string;
```

Comment:
This is more accurate. The field will always be there, but will be null.

```suggestion
    url: string | null;
```